pipelines/performance-tests-pipeline.yml

```diff
@@ -410,7 +410,7 @@ jobs:
     fieldwork-adapter,
     notify-processor,
     uac-qid-service,
-    pubsubsvc,
+    pubsub-adaptersvc,
```

Comment:
For consistency with the other pipeline I'd rather this was just `pubsub-adapter`